src/src/utils/auth.js

```diff
@@ -0,0 +1,19 @@
+import Cookies from 'js-cookies';
+import decode from 'jwt-decode';
+
+const Tokenkey = 'vue_admin_identity_token';
+const StateKey = 'vue_admin_identity_state';
+const TenantApplicationIdKey = 'tenant_application_id';
+
+export function getSolutionId() {
+
+}
+export function getUsername() {
+  return Cookies.get('vue-username');
+}
+export function getLanguageCode() {
+  return Cookies.get('language');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
  return Cookies.get("language");
```